src/DIRAC/DataManagementSystem/Agent/FTS3Agent.py

```diff
@@ -54,6 +54,11 @@
 # Lifetime in seconds of the proxy we download for submission
 PROXY_LIFETIME = 43200  # 12 hours
 
+# Instead of querying many jobs at once,
+# which maximizes the possibility of race condition
```

Comment:
Just wondering if you want to rephrase this now that the race condition is fixed